backend/api/views.py

```diff
@@ -0,0 +1,8 @@
+from rest_framework import viewsets
+from api.serializers import RouteSerializer
+from api.models import Route
+
+
+class RouteViewSet(viewsets.ModelViewSet):
+    queryset = Route.objects.all()
+    serializer_class = RouteSerializer
```

Comment:
missing newline

Reply:
The file was moved to a different directory, I didn't add any code ;). Let me fix this.